### PR TITLE
Change _instances from Array to Set

### PR DESCRIPTION
**Pull request by anthonykoerber**
peruse at your leisure - this is a follow up to our `Set` conversation yesterday.
@dlasky @shuwen @jcmoore

### CODE REVIEW

File: src/mm-list-item/mm-list-item.js [hunk 1]
```diff
@@ -78,9 +78,9 @@
 			}
 		},
 
-		elementResize: function() {
-			this.debounce("update-title", this.updateTitle, 0);
-		},
+		// elementResize: function() {
```
Comment (dlasky) [inline src/mm-list-item/mm-list-item.js]: why comment? lets just strip it

File: src/shared/behaviors/resizable.html [hunk 2]
```diff
@@ -8,38 +8,41 @@
...
 	}
 
 	function _removeInstance(instance) {
-		_instances.filter(function(meta) {
-			return meta.instance === instance;
-		}).forEach(function(target) {
-			_instances.splice(_instances.indexOf(target), 1);
-			_il--;
+		var metaArr = Array.from(_instances);
+		var meta = metaArr.filter(function(item){
```
Comment (dlasky) [inline src/shared/behaviors/resizable.html]: filter will return an array, so this deletion is likely to fail

File: src/shared/behaviors/stackable.html [hunk 3]
```diff
@@ -12,19 +12,19 @@
...
 			max:2*_w,
-			items:[],
+			items: new Set(),
 			topIndex:1*_w
 		},
 		ui:{
 			base:2*_w,
 			max:3*_w,
-			items:[],
+			items: new Set,
```
Comment (shuwen) [inline src/shared/behaviors/stackable.html]: forgot `()` here

**Comment by anthonykoerber** [comment]
green button time, gentlemen? @shuwen @jcmoore @dlasky

**Comment by shuwen** [comment]
:+1:

**Comment by dlasky** [comment]
ye

**Pull request merged**